lib/translator/inquiry.cc

```diff
@@ -187,6 +187,56 @@ void TranslateUnitSerialNumberVpd(
   memcpy(&buffer[sizeof(result)], &product_serial_number, result.page_length);
 }
 
+void TranslateBuildBlockLimitsVpd(
+    const nvme_defs::IdentifyControllerData &identify_controller_data,
+    absl::Span<uint8_t> buffer) {
+  uint32_t max_transfer_length =
+      identify_controller_data.mdts ? 1 << identify_controller_data.mdts : 0;
```

Comment:
can we add a comment somewhere that the mdts represents the size as 2^n. I was a little confused by this bit shift initially. Also mdts is at most 2^8, while the size of max_transfer_length is 4 bytes. Is there a concern of overflow?

Reply:
welll i hope nobody expects a page length of 2^2^8
good point though i'll look into it

Reply:
I'll add a bounds check